test/Microsoft.Azure.Devices.Edge.Test/helpers/TestResultCoordinatorUtil.cs

```diff
@@ -88,7 +88,7 @@ public static async Task ValidateResultsAsync()
             bool isPassed = (bool)JArray.Parse(jsonstring)[0]["IsPassed"];
             if (!isPassed)
             {
-                Log.Information("Test Result Coordinator response: {Response}", jsonstring);
+                Log.Verbose("Test Result Coordinator response: {Response}", jsonstring);
```

Comment:
If it is not passing I think it would be helpful to see the TRC report.

Reply:
Sure, and it's captured in the logfile (published as an artifact). The benefit here is that the output in DevOps stays relatively clean.

Reply:
This makes sense. Although for the proxy tests though we cannot see the devops logfile output.

Reply:
Yes, proxy tests are a problem right now. Hopefully that's temporary. If you need to investigate the proxy tests specifically, you can set `"verbose": true` in context.json. Actually I can add that in `e2e-setup.yaml`.

Reply:
FYI I updated `e2e-setup.yaml` to set `"verbose": true` in context.json by default for the proxy job.

Reply:
BTW @and-rewsmith I noticed in your recent PR you completely removed this line. Is there another way that the TRC response is being reported now?

Reply:
Ah shoot I accidentally removed this because I thought it was added with some custom debug logs I had. We should add this line back.

Reply:
I'll add it back

Reply:
Added